NOTICE.txt

```diff
@@ -1494,6 +1494,50 @@ Files covered:
 spyder/images/winpython.svg
 
 
+-------------------------------------------------------------------------------
+
+
+
+OOjs UI MediaWiki lib
+---------------------
+
+Copyright (c) 2011-2018 OOUI Team and other contributors.
+
+
+Principal Authors:
+Bartosz Dziewoński <matma.rex@gmail.com>
+Ed Sanders <esanders@wikimedia.org>
+James D. Forrester <jforrester@wikimedia.org>
+Kirsten Menger-Anderson <kmenger@wikimedia.org>
+Kunal Mehta <legoktm@gmail.com>
+Moriel Schottlender <moriel@gmail.com>
+Prateek Saxena <prtksxna@gmail.com>
+Roan Kattouw <roan@wikimedia.org>
+Rob Moen <rmoen@wikimedia.org>
+Timo Tijhof <krinklemail@gmail.com>
+Trevor Parscal <trevor@wikimedia.org>
+Volker E. <volker.e@wikimedia.org>
+
+Source/Download: https://phabricator.wikimedia.org/diffusion/GVED/browse/master/lib/oojs-ui/themes/mediawiki/images/icons/
```

Comment:
This link's not current (as of the time you downloaded the image), as the files were moved in the repo. Keep this as the source, but you can add the Wikimedia commons link, https://commons.wikimedia.org/wiki/File:OOjs_UI_icon_regular-expression.svg , as the Download link since that's where you actually downloaded it from.